engine/e2e-test/test_cli_model_import.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from test_runner import run
+
+class TestCliModelImport:
+
+    @pytest.mark.skipif(True, reason="Expensive test. Only test when you have local gguf file.")
+    def test_model_import_should_be_success(self):
+
+        exit_code, output, error = run(
+            "Pull model", ["models", "import", "--model_id","test_model","--model_path","/path/to/local/gguf"],
+            timeout=None
+        )
+        assert exit_code == 0, f"Model import failed failed with error: {error}"
```

Comment:
hmm, if running in isolation, this test will failed, right?

Reply:
Yes, this test is used only if logics change for the model-import command.